crates/pop-contracts/src/call.rs

```diff
@@ -225,11 +250,9 @@ mod tests {
 			suri: "//Alice".to_string(),
 			execute: false,
 		};
-		let call = set_up_call(call_opts).await;
-		assert!(call.is_err());
-		let error = call.err().unwrap();
-		assert_eq!(error.root_cause().to_string(), "Failed to find any contract artifacts in target directory. \nRun `cargo contract build --release` to generate the artifacts.");
-
+		assert!(
+			matches!(set_up_call(call_opts).await, Err(Error::AnyhowError(message)) if message.root_cause().to_string() == "Failed to find any contract artifacts in target directory. \nRun `cargo contract build --release` to generate the artifacts.")
```

Comment:
In the future we should ideally prompt to run builds with pop.

Reply:
It makes sense. To keep it consistent we should do the same for `pop up contract`, so probably better be in a separate PR: https://github.com/r0gue-io/pop-cli/issues/356

Reply:
:100: